examples/base-ui-nextjs-tailwind-ts/README.md

```diff
@@ -16,17 +16,8 @@ cd base-ui-nextjs-tailwind-ts
 Install dependencies and run:
 
 ```bash
-# npm
 npm install
 npm run dev
-
-# yarn
-yarn
-yarn dev
-
-# pnpm
-pnpm install
-pnpm dev
 ```
```

Comment:
Beginners uses npm, the rest might use yarn or pnpm but are not beginnings, so don't really need these.